### DISCUSSION TITLE
VHPI support in windows

### DISCUSSION THREAD

**Discussion by cameronweston** [nickg/nvc#1076] [Q&A]
Does nvc work with cocotb in Windows? I've used it exclusively in Linux. A co-worker of mine installed cocotb and nvcin windows. Cocotb threw an error saying it couldn't find an .dll for nvc. Looking in the indicated folder, I can see DLLs for most other simulators but not nvc.

**Comment by nickg** [comment]
It should work: all the VHPI regression tests run in Windows. I think cocotb's build system is missing rules to build it there though.
